hazelcast/src/main/java/com/hazelcast/client/impl/protocol/task/AbstractMessageTask.java

```diff
@@ -114,6 +113,7 @@ private void initializeAndProcessMessage() {
         if (!node.joined()) {
             throw new HazelcastInstanceNotActiveException("Hazelcast instance is not ready yet!");
         }
+        parameters = decodeClientMessage(clientMessage);
```

Comment:
@pveentjer 
Or since we are just in bug fix period. We should postpone the renaming even further ?